### ISSUE TITLE
 Denon Receivers Not Showing Up in Apple Home App

### ISSUE THREAD

**Issue by supernitin** [grzegorz914/homebridge-denon-tv#266]
After adding my Denon receivers to Homebridge, they show up in Homebridge but do not appear in the Apple Home App. I have followed all recommended steps including ensuring correct IP addresses and ports, simplifying the configuration, and enabling sensorPower. Despite this, the accessories still do not appear in the Home app. Other Homebridge accessories are functioning correctly.

Configuration:
`{
  "bridge": {
    "name": "Homebridge",
    "username": "CC:22:3D:E3:CE:30",
    "port": 51826,
    "pin": "123-45-678"
  },
  "accessories": [],
  "platforms": [
    {
      "platform": "DenonTv",
      "devices": [
        {
          "name": "Main Floor Ceiling Speakers",
          "host": "192.168.68.133",
          "port": 8080,
          "generation": 1,
          "zoneControl": 0,
          "sensorPower": true,
          "masterPower": true,
          "inputsDisplayOrder": 0,
          "enableDebugMode": true
        },
        {
          "name": "Deck Speakers",
          "host": "192.168.68.133",
          "port": 8080,
          "generation": 1,
          "zoneControl": 1,
          "sensorPower": false,
          "masterPower": false
        },
        {
          "name": "Bed Ceiling Speakers",
          "host": "192.168.68.104",
          "port": 8080,
          "generation": 1,
          "zoneControl": 0,
          "sensorPower": false,
          "masterPower": false
        },
        {
          "name": "Basement Ceiling Speakers",
          "host": "192.168.68.104",
          "port": 8080,
          "generation": 1,
          "zoneControl": 1,
          "sensorPower": false,
          "masterPower": false
        }
      ]
    }
  ],
  "_bridge": {
    "username": "0E:11:F9:21:43:80",
    "port": 44465
  }
}
`
Details:

Models: Denon AVR-S970H and AVR-S760H
Zones:
AVR-S970H: Main Floor Ceiling Speakers (zone 1), Deck Speakers (zone 2)
AVR-S760H: Bed Ceiling Speakers (zone 1), Basement Ceiling Speakers (zone 2)
Other Accessories: Other Homebridge accessories are functioning and visible in the Apple Home app.
Network: All devices are on the same network, and HomeKit permissions are correctly set up.
Steps Taken:

Verified IP addresses and ports are correct.
Used curl to confirm connectivity.
Simplified the configuration to a single device with sensorPower and masterPower enabled.
Restarted Homebridge and re-paired the child bridge with the Home app.
Checked Homebridge logs for errors or warnings.
Ensured Homebridge and the Denon TV plugin are up to date.
Ensured all devices are on the same network and HomeKit permissions are correct.
Logs:
Here are some log entries indicating the devices are being initialized but not exposed to HomeKit:
`[6/3/2024, 8:49:59 AM] [homebridge-denon-tv] Initializing platform accessory 'Main Floor Ceiling Speakers'...
[6/3/2024, 8:50:00 AM] [homebridge-denon-tv] Finished initializing platform accessory 'Main Floor Ceiling Speakers'.
`
Expected Behavior:
The accessories should appear in the Apple Home app after pairing the child bridge.

Actual Behavior:
The accessories do not appear in the Apple Home app despite showing up in Homebridge.

Request for Assistance:
Any guidance on resolving this issue would be greatly appreciated. Please let me know if additional information or logs are needed.

Thank you!

**Comment by grzegorz914** [comment]
https://github.com/grzegorz914/homebridge-denon-tv/wiki#adding-to-homekit

**Comment by wx2junior** [comment]
I had this occur as well, what ended up fixing it for me was:

1. Completely remove bridges etc from the home kit app on the device... (3 dots then home settings > Home Hubs & Bridges removed anything related to denon from home app)
2. Within the homebridge UI > 3 dots > Settings scroll down to the reset section and make sure everything related to the homebridge-denon-tv child hub or accessories was removed from the "Unpair Bridges / Cameras / TVs / External Accessories" list... 

Once I did the above, when I went into the home app and did the add accessory, the Receiver showed up where expected during the add accessory flow.
